md/two-main-types-of-deployment.md

```diff
@@ -29,6 +30,20 @@ It is fast because the Bonita Portal and the Bonita Engine run on the same JVM a
 **Drawbacks**
 
 * may not be adapted to some architecture constraints
+## Bonita Portal + Bonita Engine running in a docker container
+
+This is a new deployment option available in the 7.10 version.
+The installation process is described [here](bonita-docker-installation.md)
+**Advantages**
+
+* The usual advantages provided by using a docker container : isolation, simplicity, scalability...
+* Works out of the box with minimal effort.
+* Is the natural solution for a cluster or cloud deployment
+
+**Drawbacks**
+
+* Probably overkill for a simple on premise single server installation
+* More complicated installation than the previous solution
```

Comment:
I don't agree.
It's not overkill: it's more a matter of personal choice. If you already have some docker, it's easier to just run docker containers.
Also It's not more complicated installation (or it should not be). That's what you said in `Advantages`

IMHO, I would only put here something like `Require the installation of Docker`.

Reply:
Agreed with @baptistemesta. I would not put **any** of those drawbacks, as they are not, for me:
* It's not overkill because docker is thought to be the same on an on-premise installation and "in the cloud".
* It's not more complicated to run

Reply:
maybe we can put as drawback that it could be harder to customize the installation... but I'm not sure